farge

```diff
@@ -169,38 +173,36 @@ main() {
         # Write the color code in the center of the image
         convert "$FNAME" -gravity Center -pointsize 10 -fill "$TEXT_COLOR" -annotate 0 "$COLOR_CODE" "$FNAME"
     fi
-    
+
     # Save the color code to clipboard
     if [ "$WAYLAND_DISPLAY" ]; then
-        # We are removing the new line char and then sending it to the clipboard
-        echo "$COLOR_CODE" | tr -d "\n" | wl-copy
+        printf %s "$COLOR_CODE" | wl-copy
     else
-        echo "$COLOR_CODE" | tr -d "\n" | xclip -selection c
+        printf %s "$COLOR_CODE" | xclip -sel c
     fi
-    
-    # Output the color code to stdout only if the user uses the --stdout flag
-    [ "$STDOUT" == true ] && echo "$COLOR_CODE"
 
-    if [ "$NOTIFICATION" == "true" ]; then
+    # Output the color code to stdout only if the user uses the --stdout flag
+    if [ "$STDOUT" == true ]; then
+        echo "$COLOR_CODE"
+    elif [ "$NOTIFICATION" == "true" ]; then
         notify-send "$COLOR_CODE" --icon="$FNAME" --expire-time="$EXPIRE_TIME"
-
     elif [ "$PREVIEW" == "true" ]; then
-        image_preview 
+        image_preview
     fi
```

Comment:
Previously this printed to stdout *and* opened the image viewer, it now only prints to stdout like it says.

(Also removed useless spaces at the end of some lines)